tests/cupy_tests/fft_tests/test_fft.py

```diff
@@ -159,6 +161,17 @@ def test_ifft(self, xp, dtype):
         return out
 
 
+# See #3757 and NVIDIA internal ticket 3093094
+def _skip_multi_gpu_bug(shape, gpus):
+    # avoid CUDA 11 bug triggered by
+    # - batch = 1
+    # - gpus = [1, 0]
+    if (cupy.cuda.runtime.runtimeGetVersion() == 11000
```

Comment:
```suggestion
    if (cupy.cuda.runtime.runtimeGetVersion() == 11200
```

Reply:
Originally I forgot to change the comparison operator, which I fixed now.

Reply:
I think you meant this?
```suggestion
    if (11000 <= cupy.cuda.runtime.runtimeGetVersion() < 11200
```

Reply:
Ah yes, totally missed that, thanks for fixing!